### ISSUE TITLE
About the first experiment

### ISSUE THREAD

**Issue by ryanbbd** [dylanrandle/denn#5]
nice job! I've tried my GAN for a long time for solving PDE, failed. But, when I try your code for the first exp experiment, I can't get the solution as yours. My val_mse and train_mse always bigger than 2. Do you update your hyper-parameter configuration? or maybe model construction?

**Comment by ryanbbd** [comment]
Just don't understand this loss term:
`wass = lambda y_true, y_pred: torch.mean(y_true * y_pred)`
Could you give me some tips, thanks a lot

**Comment by dylanrandle** [comment]
Hi there,

1. The results are very sensitive to initialization (with random seed). But moreover, difficult computing architectures can differ in how they handle this randomness (e.g. I have observed differences between MacOS and Linux environments). This could be the case for you. I would try various random seeds to see if one of them converges. We have no guarantees for convergence with GANs, unfortunately.

2. The loss term is the Wasserstein loss function (where the labels are -1 and 1 instead of 0 and 1 as with vanilla GAN), see here: https://arxiv.org/abs/1701.07875.

**Comment by dylanrandle** [comment]
Going to close this is this is not a new/unknown issue, but one that has been observed and is detailed in the documentation (i.e. the importance of the platform).
